index.js

```diff
@@ -128,7 +128,7 @@ var appServer = function(config) {
         postRequest: self.config.postRequest
       });
 
-      var endpoint = path.posix.join(normalizedRoot, pkg.name);
+      var endpoint = path.posix.join(normalizedRoot, app.name);
       self.log("   loaded app [" + pkg.name + "] at endpoint: " + endpoint);
```

Comment:
If you feel motivated, you could write a test for this using `sinon`, like https://github.com/alexa-js/alexa-app/blob/b30997f1e2fce2302bf1b04e855b19837edf5e8f/test/test_alexa_integration_express.js#L51